Tests/Fixtures/Generated/Colors/swift4/defaults-customName.swift

```diff
@@ -35,18 +35,32 @@ internal struct XCTColors {
 
 // MARK: - Implementation Details
 
-// swiftlint:disable operator_usage_whitespace colon
+// swiftlint:disable operator_usage_whitespace
```

Comment:
I wonder why is `operator_usage_whitespace` still useful? I don't see in the remaining code any violation of that rule, is it? 🤔

Reply:
Must just be a relic of the past 🙂 I'll remove it.